sway-lib-std/src/storage.sw

```diff
@@ -134,14 +133,13 @@ impl<V> StorageVec<V> {
     }
 
     /// Removes the last element of the vector and returns it, None if empty
-    #[storage(read, write)]
-    pub fn pop(self) -> Option<V> {
+    #[storage(read, write)]pub fn pop(self) -> Option<V> {
```

Comment:
Most of the languages I have used in the past prefer to keep attributes as a separate line, which at least to me looks a bit more readable. I guess this is fine too, just wondering if this was automatically or manually formatted and if this is way we prefer to format them so I can be consistent in the future.

Reply:
Yup, this is very ugly but that's what the current formatter does. Luckily, the new formatter will not have this issue. So yeah, this happened because I ran the formatter on all of the standard library which generally should be done with every change and should probably be checked in CI (I think there's an issue about that somewhere)